wallet/chainntfns.go

```diff
@@ -148,6 +148,9 @@ func (w *Wallet) addReceivedTx(tx *btcutil.Tx, block *txstore.Block) error {
 			if err != nil {
 				return err
 			}
+			if err := w.markAddrsUsed(txr); err != nil {
```

Comment:
This belongs in a different pr (or wait for intwtxmgr, which doesn't have this bug)